cmd/nodecmd/dynamic_ips.go

```diff
@@ -5,40 +5,42 @@ package nodecmd
 import (
 	"context"
 	"fmt"
+	"strings"
 
 	awsAPI "github.com/ava-labs/avalanche-cli/pkg/cloud/aws"
 	gcpAPI "github.com/ava-labs/avalanche-cli/pkg/cloud/gcp"
 
-	"github.com/ava-labs/avalanche-cli/pkg/constants"
-
 	"github.com/ava-labs/avalanche-cli/pkg/ansible"
-
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
 	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/utils"
 	"github.com/ava-labs/avalanche-cli/pkg/ux"
 )
 
-func getNodesWoEIPInAnsibleInventory(clusterNodes []string) []models.NodeConfig {
-	nodesWoEIP := []models.NodeConfig{}
+func getNodesWithDynamicIP(clusterNodes []string) ([]models.NodeConfig, error) {
+	nodesWithDynamicIP := []models.NodeConfig{}
 	for _, node := range clusterNodes {
 		nodeConfig, err := app.LoadClusterNodeConfig(node)
 		if err != nil {
-			continue
+			return nil, err
 		}
-		if nodeConfig.ElasticIP == "" {
-			nodesWoEIP = append(nodesWoEIP, nodeConfig)
+		if !nodeConfig.UseStaticIP {
+			nodesWithDynamicIP = append(nodesWithDynamicIP, nodeConfig)
 		}
 	}
-	return nodesWoEIP
+	return nodesWithDynamicIP, nil
 }
 
-func getPublicIPForNodesWoEIP(nodesWoEIP []models.NodeConfig) (map[string]string, error) {
-	lastRegion := ""
-	var ec2Svc *awsAPI.AwsCloud
-	var err error
+func getPublicIPsForNodesWithDynamicIP(nodesWithDynamicIP []models.NodeConfig) (map[string]string, error) {
 	publicIPMap := make(map[string]string)
-	var gcpCloud *gcpAPI.GcpCloud
-	ux.Logger.PrintToUser("Getting Public IPs for nodes without static IPs ...")
-	for _, node := range nodesWoEIP {
+	var (
+		err        error
+		lastRegion string
+		ec2Svc     *awsAPI.AwsCloud
+		gcpCloud   *gcpAPI.GcpCloud
+	)
+	ux.Logger.PrintToUser("Getting Public IPs for nodes with dynamic IPs ...")
```

Comment:
nit: should be imho `ux.Logger.PrintToUser("Getting Public IP(s) for node(s) ...")`